verification/curator-service/ui/src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,180 @@
+    const location = useLocation();
+
+    const [createNewButtonAnchorEl, setCreateNewButtonAnchorEl] =
+        useState<Element | null>();
+    const [selectedMenuIndex, setSelectedMenuIndex] = React.useState<number>();
+    const user = useAppSelector(selectUser);
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const openCreateNewPopup = (event: any): void => {
+        setCreateNewButtonAnchorEl(event.currentTarget);
+    };
+
+    const closeCreateNewPopup = (): void => {
+        setCreateNewButtonAnchorEl(undefined);
+    };
+
+    const hasAnyRole = useCallback(
+        (requiredRoles: string[]): boolean => {
+            if (!user) {
+                return false;
+            }
+            return user?.roles?.some((r: string) => requiredRoles.includes(r));
+        },
+        [user],
+    );
+
+    const menuList = useMemo(
+        () =>
+            user
+                ? [
+                      {
+                          text: 'Line list',
+                          icon: <ListIcon />,
+                          to: { pathname: '/cases', search: '' },
+                          displayCheck: (): boolean => true,
+                      },
+                      {
+                          text: 'Sources',
+                          icon: <LinkIcon />,
+                          to: '/sources',
+                          displayCheck: (): boolean => hasAnyRole(['curator']),
+                      },
+                      {
+                          text: 'Uploads',
+                          icon: <PublishIcon />,
+                          to: '/uploads',
+                          displayCheck: (): boolean => hasAnyRole(['curator']),
+                      },
+                      {
+                          text: 'Manage users',
+                          icon: <PeopleIcon />,
+                          to: '/users',
+                          displayCheck: (): boolean => hasAnyRole(['admin']),
+                      },
+                  ]
+                : [],
+        [hasAnyRole, user],
+    );
+
+    useEffect(() => {
+        const menuIndex = menuList.findIndex((menuItem) => {
+            const pathname =
+                typeof menuItem.to === 'string'
+                    ? menuItem.to
+                    : menuItem.to.pathname;
+            return pathname === location.pathname;
+        });
+        setSelectedMenuIndex(menuIndex);
+    }, [location.pathname, menuList]);
+
+    return (
+        <Drawer
+            className={classes.drawer}
+            variant="persistent"
+            anchor="left"
+            open={drawerOpen}
+            data-testid="sidebar"
+            classes={{
+                paper: classes.drawerPaper,
+            }}
+        >
+            <div className={classes.drawerContents}>
+                <div className={classes.drawerHeader}></div>
+                <Typography className={classes.covidTitle}>COVID-19</Typography>
```

Comment:
I think this needs to be a parameter for "disease title = COVID-19", not hard-coded. One of the goals for G.h is we should be able to quickly start up for a new disease.